Campus-iOS/LoginComponent/Views/TokenConfirmationView.swift

```diff
@@ -276,3 +383,9 @@ extension Bundle {
         }
     }
 }
+
+enum TokenState {
```

Comment:
Maybe also move this into some extension?

Reply:
✅